example/scalalib/testing/6-test-group-stealing/build.mill

```diff
@@ -0,0 +1,39 @@
+// This example shows how to use test grouping in combination with test stealing.
+
+//// SNIPPET:BUILD1
+package build
+import mill._, scalalib._
+
+object foo extends ScalaModule {
+  def scalaVersion = "2.13.8"
+  object test extends ScalaTests {
+    def ivyDeps = Seq(ivy"com.lihaoyi::utest:0.8.5")
+    def testFramework = "utest.runner.Framework"
+
+    def testForkGrouping = discoveredTestClasses().grouped(10).toSeq
+    def testEnableWorkStealing = true
+  }
+}
+
```

Comment:
This needs a more verbose explanation about how `testForkGrouping` and `testEnableWorkStealing` interact